KeyboardHelperTests/Tests/KeyboardHelperTests.swift

```diff
@@ -9,6 +9,93 @@
 import XCTest
 @testable import KeyboardHelper
 
+class ShowSpyDelegate : KeyboardNotificationDelegate {
```

Comment:
I guess you're right. It just felt disconnected from the code to me, but leave it as it is, it's fine :)